doc/RELEASE_WALKTHROUGH.rst.txt

```diff
@@ -1,18 +1,39 @@
-This file contains a walkthrough of the NumPy 1.12.0 release on Fedora Linux.
+This file contains a walkthrough of the NumPy 1.14.5 release on Linux.
 The commands can be copied into the command line, but be sure to
-replace 1.12.0 by the correct version.
+replace 1.14.5 by the correct version.
 
 Release  Walkthrough
 ====================
 
-Building the release
---------------------
+Backport Pull Requests, Write Release Note
+---------------------------------------------
+
+Any fixes for the release must be backported to the maintenance/1.14.x branch.
```

Comment:
This is not quite correct. Only important fixes need backporting; things that can wait should wait. There's some judgement involved here, but e.g. backporting doc typo fixes should not be done.